torchtune/models/llama3_2_vision/_model_builders.py

```diff
@@ -41,6 +41,7 @@ def llama3_2_vision_11b(
     Returns:
         DeepFusionModel: Instantiation of the Llama 3.2 Vision 11B model
     """
+    assert image_size % 14 == 0, "image_size must be divisible by 14, the encode patch size"
```

Comment:
did you mean encoder or encode?

Also, should it be ValueError, or assert? I think that normally we prefer to raise ValueError

Reply:
just saw Rafi's comment :P